### PR TITLE
Add latest EasyBuild release to NESSI/2022.11

### PR DESCRIPTION
**Pull request by trz42**
Another demo PR for illustrating the use of the EESSI bot.

### CODE REVIEW

**Comment by trz42** [comment]
Adding a comment. (check if bot instances receive event)

**Comment by trz42** [comment]
Next attempt.

**Comment by trz42** [comment]
One more.

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -459,6 +459,18 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
 #cat $($EB --last-log)
 #check_exit_code $exit_code "${ok_msg}" "${fail_msg}"
 
+## add latest EasyBuild to stack
+echo ">> Adding latest EasyBuild to stack. So far we have:"
+module --ignore_cache avail EasyBuild 2>&1 | grep EasyBuild
```
Comment (poksumdo) [inline EESSI-pilot-install-software.sh:464]: Clean up before merge.

File: EESSI-pilot-install-software.sh [hunk 2]
```diff
@@ -459,6 +459,18 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
...
+ok_msg="Latest EasyBuild got installed ... great!"
+fail_msg="Installation of latest EasyBuild failed! Disappointed."
+$EB --robot --install-latest-eb-release
+exit_code=$?
+# get what versions are available now
+echo ">> Now, we have:"
+module --ignore_cache avail EasyBuild 2>&1 | grep EasyBuild
```
Comment (poksumdo) [inline EESSI-pilot-install-software.sh:471]: Cleanup before merge.

**Review by poksumdo — CHANGES_REQUESTED** [review]
Overall looks good.

Branch needs to be 
- [x] sync'ed,
- [ ] rebuilt, and
- [ ] reingested before it can be merged. That's because it was built not against the HEAD version of `nessi.no-2022.11-dev`.

Note, reingestion might require a reconfiguration of the bot instances to upload the `latest` tarballs (see option `upload_policy`).

- [x] Also additional printouts shall be removed or the output of the job would include unnecessary information.

**Comment by trz42** [comment]
Oh, packages are not really rebuilt because they are already in the stack. Instead of forcing to rebuild we remove EasyBuild from the stack (by rolling back the transactions on the Stratum0).

**Review by poksumdo — APPROVED** [review]

**Comment by trz42** [comment]
Only tarballs for NESSI/2022.11 will be ingested. Those for EESSI/2021.12 are rejected (because here we just test the builds for EESSI/2021.12 but we don't ingest).

**Comment by trz42** [comment]
All that needed to get ingested is available via CernVM-FS at a client. Ready to merge PR.

**Pull request merged**